health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/RichSummary.js

```
@@ -0,0 +1,161 @@
+  return (
+    <div className={`digit-dss-insight-card-difference ${data?.insight?.indicator === "upper_green" ? "increase" : "decrease"} metric-card`}>
+      <Icon
+        type={data?.insight?.indicator === "upper_green" ? "arrow-upward" : "arrow-downward"}
+        iconColor={data?.insight?.indicator === "upper_green" ? "#00703C" : "#D4351C"}
+        width="1.5rem"
+        height="1.5rem"
+        className="digit-dss-insight-icon"
+      />
+      {insight?.[0] &&
+        `${Digit.Utils.dss.formatter(insight[0], "number", value?.denomination, true, t)}% ${t(
+          Digit.Utils.locale.getTransformedLocale("DSS" + insight?.[1] || "")
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)